examples/Backtesting/botorch_analytical.py

```diff
@@ -119,13 +107,24 @@
 results = simulate_scenarios(
     scenarios,
     WRAPPED_FUNCTION,
-    batch_size=3,
+    batch_size=BATCH_SIZE,
     n_doe_iterations=N_DOE_ITERATIONS,
     n_mc_iterations=N_MC_ITERATIONS,
 )
 
-# The following lines plot the results and save the plot in run_analytical.png
+# We use the plotting utility to create plots.
 
-sns.lineplot(data=results, x="Num_Experiments", y="Target_CumBest", hue="Scenario")
-plt.gcf().set_size_inches(24, 8)
-plt.savefig("./run_analytical.png")
+path = Path(sys.path[0])
+figure = sns.lineplot(
```

Comment:
```suggestion
ax = sns.lineplot(
```

ax is the most common term used for an Axis object, in any case a figure is an entirely different object type and we shouldnt mix that nomenclature here